site/snippets/capsule.ts

```diff
@@ -0,0 +1,27 @@
+import { WalletClientSigner, type SmartAccountSigner } from "@alchemy/aa-core";
+import Capsule, { createCapsuleViemClient, Environment } from "@usecapsule/web-sdk";
+import { http } from "viem";
+
+
+// get an API Key by filling out [this form](TODO)
+const CAPSULE_API_KEY = null
+
+// Capsule's viem client supports custom chains and providers
+// Here we've included Alchemy's sepolia provider
+const CHAIN = "sepolia"
```

Comment:
this chain has to be an import from `viem/chains` https://github.com/capsule-org/web-sdk/blob/main/examples/src/index.tsx#L18 and https://github.com/capsule-org/web-sdk/blob/main/examples/src/index.tsx#L135-L138